flight_planning/v1/flight_planning.yaml

```diff
@@ -0,0 +1,371 @@
+          enum: [Planned, ReadyToFly, Rejected, Failed, NotSupported]
+          example: Planned
+        notes:
+          description: >-
+            Human-readable explanation of the observed result.  This explanation
+            may be made available to a human reviewing the test results, and
+            ideally should explain why an undesirable result was obtained.  For
+            instance, if the injection attempt Failed, then these notes may
+            indicate that the attempt failed because the DSS indicated 400 to a
+            valid request (perhaps also including the valid request as proof).
+          type: string
+          example: Requested flight intersected operational intent c036326c-c97b-4926-bf9f-c60dc83d2b57
+        includes_advisories:
+          description: >-
+            Indication of whether any advisories or conditions were provided to the user along with the result of this
+            flight planning attempt.
+            
+              - `Unknown`: It is unknown or irrelevant whether advisories or conditions were provided to the user
+
+              - `Yes`: At least one advisory or condition was provided to the user.
+            
+              - `No`: No advisories or conditions were provided to the user.
+          type: string
+          enum:
+            - Unknown
+            - Yes
+            - No
+          default: Unknown
+          example: No
+
+    DeleteFlightPlanResponse:
+      type: object
+      required:
+        - result
+      properties:
+        result:
+          description: >-
+            The result of attempted flight plan cancellation/closure.
+
+              - `Closed`: The flight plan was closed successfully by the USS and is now out of the UTM system.
+
+              - `Failed`: The flight plan could not be closed successfully by the USS.
+          enum: [Closed, Failed]
+          example: Failed
+        notes:
+          description: >-
+            Human-readable explanation of the observed result.
+          type: string
+          example: DSS was unreachable when attempting to delete operational intent reference
+
+    ClearAreaRequest:
+      type: object
+      required:
+        - request_id
+        - extent
+      properties:
+        request_id:
+          description: >-
+            Unique string identifying this request.  If a second request with an
+            identical ID is received, the USS may return the same response from
+            the previous operation rather than attempting to clear the area
+            again (the USS may also attempt to clear the area again).
+          type: string
+        extent:
+          description: >-
+            The USS should cancel and remove any flight plan where any part of that
+            flight plan intersects this area.
+          anyOf:
+            - $ref: './astm_f3548_21.yaml#/components/schemas/Volume4D'
+    ClearAreaOutcome:
+      type: object
+      required:
+        - timestamp
```

Comment:
Removed `timestamp`; thanks.  Since `success` has a default value of false, it is well-defined if not specified.  Some code/server generators (gRPC/protobuf, specifically) do not specify the values of fields that have the default value of the data type (e.g., a false boolean would not be specified), so to promote maximum usability even in the face of very questionable design decisions, the practice I try to follow in general is that all primitive-data-type fields are optional with default values equal to the default value of the data type.

Reply:
> so to promote maximum usability even in the face of very questionable design decisions, the practice I try to follow in general is that all primitive-data-type fields are optional with default values equal to the default value of the data type.

That makes a lot of sense, thanks for the explanation :)